pkg/namespacestore/namespacestore.go

```diff
@@ -174,6 +175,24 @@ func CmdCreateAzureBlob() *cobra.Command {
 	return cmd
 }
 
+// CmdCreateNSFS returns a CLI command
+func CmdCreateNSFS() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "nsfs <namespace-store-name>",
+		Short: "Create nsfs namespace store",
+		Run:   RunCreateNSFS,
+	}
+	cmd.Flags().String(
+		"fs-backend", "",
```

Comment:
I assume this is only in the initial PR?
Because I think that there should be strict types for that, does it make sense?

Reply:
added an enum validation to the namespace store types, thank you.